interfaces/acados_c/ocp_qp_interface.h

```diff
@@ -63,25 +63,39 @@ typedef enum {
     PARTIAL_CONDENSING_HPIPM,
 #ifdef ACADOS_WITH_HPMPC
     PARTIAL_CONDENSING_HPMPC,
+#else
+    PARTIAL_CONDENSING_HPMPC_NOT_AVAILABLE,
```

Comment:
I think this should work fine, ATM I don't see cases where this would create issues.